spec/support/helpers/model_spec_helpers.rb

```diff
@@ -1,8 +1,10 @@
 module ModelSpecHelpers
-  def expect_location_answer(answer, params)
+  def expect_location_answer(response, answer, params)
+    response.save
     answer.save
+
```

Comment:
I'm not certain about this response.save and answer.save, and what it means for how we're handling saving responses and the response tree. If I remove answer.save, then spec/models/results/geopoint_processing_spec.rb:223 fails (and no other specs fail). It updates a location with less data.